guides/common/modules/con_migrating-project-to-a-new-el-system.adoc

```diff
@@ -1,25 +0,0 @@
-[id="migrating-project-to-a-new-el-system_{context}"]
```

Comment:
I am unsure if we should really drop this procedure. I feel like this could be part of a broader backup and restore concept; maybe to clone prod systems?